css/main.css

```diff
@@ -0,0 +1,446 @@
+* {
+    padding: 0;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: #1b1b1b;
+}
+
+a {
+    text-decoration: none;
+}
+
+header {
+    text-align: left;
+    padding-inline: 10%;
+    outline: 3px solid rgba(100, 100, 100, 0.459);
+}
+
+
+nav {
+    height: 10vh;
+    line-height: 10vh;
+    white-space: nowrap;
+}
+
+.upper-nav-bar {
+    display: flex;
+    align-items: center;
+    justify-content: left;
+    flex-direction: row;
+}
+
+.upper-nav-bar > li:first-child{
+    margin-left: calc( 50% - 3em );
+}
+
+.upper-nav-bar > li > a {
+    margin: auto;
+    padding-inline: 1em;
+    color:whitesmoke;
+}
+
+.first-screen {
+    display: flex;
+    height: 70vh;
+    background: rgb(36, 31, 56);
+    padding-inline: 20%;
+    justify-content: center;
+    text-align: center;
+    margin-top:-10vh;
+    padding-top: 10vh;
+    flex-direction: column;
+}
+
+.first-screen > div {
+    margin: auto;
+    color:whitesmoke;
+}
+
+.first-screen > div:first-child {
```

Comment:
Apply to first `.first-screen__horizontal-block` modifier `.first-screen__horizontal-block_margin-top-20px`.